pages/accountLists.page.tsx

```diff
@@ -40,9 +40,12 @@ export const getServerSideProps: GetServerSideProps = async ({
   const token = session?.user.token;
 
   if (!token) {
-    res.writeHead(302, { Location: '/' });
-    res.end();
-    return { props: {} };
+    return {
+      redirect: {
+        destination: '/login',
+        permanent: false,
+      },
+    };
```

Comment:
In the docs, I found a better/updated way to handle redirects. If the session was expired or not present, the ```res.writeHead``` would consistently throw errors.